guix/README.org

```diff
@@ -16,9 +16,9 @@ source tree is not meant to be a package ... or some such), =Gerbil=
 and =Gambit= in general rely on a git tree to tell them certain
 things.
 
+There is *Fare*'s nix package which helped.
 
 https://github.com/MuKnIO/nixpkgs/tree/dbee0d9a81487c79d7e11a9e3e3da589ac5c0fc6/pkgs/development/compilers/gerbil
```

Comment:
Maybe instead point to the upstream release and/or the devel branch on mukn?
https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/compilers/gerbil
https://github.com/muknio/nixpkgs/tree/devel/pkgs/development/compilers/gerbil

Reply:
done